src/ProductAnalytics/Context.tsx

```diff
@@ -6,22 +6,27 @@ import React, {
   useContext,
 } from "react"
 import Matomo from "react-native-matomo-sdk"
+
 import { StorageUtils } from "../utils"
-import { actions } from "./index"
 import { useConfigurationContext } from "../ConfigurationContext"
+import * as analyticsClient from "./analyticsClient"
```

Comment:
```suggestion
import * as AnalyticsClient from "./analyticsClient"
```

Reply:
I like the idea of the AnalyticsClient, what do you think about passing it in as a prop to the provider?

so something like:
```typescript
<ProductAnalyticsProvider analyticsClient={Matomo}>
   ...consumers
</ProductAnalyticsProvider>
```